### ISSUE TITLE
The installation guide should be in the Sphinx docs

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#5982]
```
http://docs.sympy.org/0.7.1/install.html redirects to http://code.google.com/p/sympy/wiki/DownloadInstallation?tm=2 , which is out of date (at least the examples are).  I think that it should be in the docs anyway, and this will also help keep things up-to-date.

Also, I have no idea if the installation guides for other systems (like Gentoo) are up-to-date or not.  I think we probably should just recommend that people install from source anyway, though (do others agree with this?), either from the latest version or from git.

So, to summarize, this page needs to be updated and moved to the Sphinx docs.
```
Original issue for #5982: http://code.google.com/p/sympy/issues/detail?id=2883
Original author: https://code.google.com/u/asmeurer@gmail.com/

**Comment by sagarpatel10** [comment]
Hey so I just joined this issue recently, and I am sort of a newbie at open source codes can anyone point me in the right direction??

**Comment by asmeurer** [comment]
I don't think this issue is relevant any more.

**Comment by sagarpatel10** [comment]
Oh ok Thanks for letting me know
